042-SAPOnAzure/Coach/01-SAP-Auto-Deployment.md

```diff
@@ -38,13 +38,13 @@ Also make sure that "./local_setup_env.sh" is executed
 
 For step 7. Make sure that the students only modify the top section indicated and not below.
 
-For step 9. The provision of E32 server and ANF provision could take long and timeout and error out. In case it happened, students need to cleanup all the resources and resource group "saprg_ophk_teamXX" and restart this step.
+For step 9. The provision of E32 server and ANF provision could take long and timeout and error out. In case it happened, students need to cleanup all the resources and resource group "saprg_ophk_squdXX" and restart this step.
```

Comment:
I think this should be `saprg_ophk_squadXX`

Reply:
Also, instead of the quotes, use the back ticks to make it monospaced like I just did in my last comment.

Reply:
Replaced quotes with back ticks.